backend/index.js

```diff
@@ -4,7 +4,14 @@ import express from "express"
 import cors from "cors"
 import routes from "./src/routes.js"
 
-mongoose.connect(process.env.MONGO_DB_URI || "mongodb://127.0.0.1:27017/ecommerce")
+mongoose
+  .connect(process.env.MONGO_DB_URI || "mongodb://127.0.0.1:27017/ecommerce")
+  .then((result) => {
+    console.log("connected to MongoDB")
```

Comment:
recuerda usar para estos console usar textos estándar y un poco mas descriptivos ej:
`[SUCCESS]: connected to MongoDB on port ${process.env.MONGO_DB_URI || "27017"}`